scripts/windows_setup.bat

```diff
@@ -0,0 +1,10 @@
+@echo off
+
+echo Setting environment variables...
+
+setx PYBAMM_USE_VCPKG ON
+setx VCPKG_DEFAULT_TRIPLET x64-windows-static-md
+setx VCPKG_FEATURE_FLAGS manifests,registries
+
+echo Environment variables have been set. Please open a new Command Prompt to see the changes.
+pause
```

Comment:
Okay, I will try this and update you

Reply:
`install_KLU_Sundials.py` isn't supported and does not work for Windows, it's a script made just for Unix-based platforms

Reply:
I wanted to clarify: currently we don't support windows users running the `install_KLU_Sundials.py` script. Adding this feature in this script would change that, and prompting more changes and maybe errors. 

I think the way it is set currently is better as it is: 
1. `install_KLU_Sundials` for linux and macos to install the libs
2. `setup_windows.bat` to setup the env vars and the installation is carried out automatically by vcpkg.

I feel this is better. What do you say?

Reply:
Yes, I feel this is better for now. We should start testing `setup_windows.bat` in CI sometime soon, though. 

A cross-platform script that handles everything from Windows to Linux would be great, but maybe we would need to switch to `vcpkg` entirely for that.

Reply:
I did try the script locally and it works. For CI though, because of the `pause` in the last line of the script, the CI fails. From what I understand, it fails to work with interactive scripts (pause is needed to get that "press any key to continue" screen for the users, otherwise it would just flash and close). So I think it would be nice that we work with env vars like we have been, or find a way to make the CI interactive (??).

Reply:
Interactive is terrible for CI. What is the pause needed for?

Reply:
pause literally pauses the script till any key is pressed to close it. It's more of a UI/UX thing for users. Otherwise when you run it, the environment vars would be set and the terminal closed automatically

Reply:
CI can't be made interactive, it's non-interactive by design :) Unless you connect to a runner via SSH. Not needed for now, but once we add it to CI somewhere, you can check for `CIBUILDWHEEL` as an environment variable and only add the `pause` there – we do something similar for the `docs` session in `noxfile.py`.